meshery

```diff
@@ -127,42 +127,77 @@ unzip ${PWD}/meshery.zip -d ${PWD}/$temporary_dir_name
 install_dir=${MESHERY_INSTALL_DIR}
 if [ -z "$install_dir" ]
 then
-    install_dir="/usr/local/bin"
+install_dir="/usr/local/bin"
 fi
 
 printf "\nInstalling mesheryctl in $install_dir.\n"
 WHOAMI=$(whoami)
 if mv ${PWD}/$temporary_dir_name/mesheryctl $install_dir/mesheryctl ; then
-  echo "mesheryctl installed."
+echo "mesheryctl installed."
 else
-  if sudo mv ${PWD}/$temporary_dir_name/mesheryctl $install_dir/mesheryctl ; then
+if sudo mv ${PWD}/$temporary_dir_name/mesheryctl $install_dir/mesheryctl ; then
     echo "Permission Resolved: mesheryctl installed using sudo permissions."
-  else
+else
     echo "Cannot install mesheryctl. Check permissions of $WHOAMI for $install_dir."
     exit 1
-  fi
+fi
 fi
 
 #Transfering permissions and ownership to client USER
 if [ "$SUDO_USER" == "" ] ;
 then
-	if chown $WHOAMI $install_dir/mesheryctl ; then
+if chown $WHOAMI $install_dir/mesheryctl ; then
     echo "permissions moved to "$WHOAMI
-  else
+else
     echo "Unable to write to $install_dir. Please verify write permission for $WHOAMI."
     exit 1
-  fi
+fi
 else
-	if chown $SUDO_USER /usr/local/bin/mesheryctl ; then
+if chown $SUDO_USER /usr/local/bin/mesheryctl ; then
     echo "permissions moved to "$SUDO_USER
-  else
+else
     echo "Unable to write to $install_dir/mesheryctl. Please verify write permission for $SUDO_USER."
     exit 1
-  fi
+fi
 fi
 
 
 printf "Removing installation files and opening Meshery..."
 rm -rf meshery.zip ${PWD}/$temporary_dir_name/
 
+
```

Comment:
@devansh42, You should move these changes towards the end of the script. i.e. after the recently introduced 
 if [ "$DEPLOY_MESHERY" == false ] ; check.